openpype/hosts/webpublisher/api/__init__.py

```diff
@@ -0,0 +1,39 @@
+import os
+import logging
+
+from avalon import api as avalon
+from avalon import io
+from pyblish import api as pyblish
+import openpype.hosts.webpublisher
+
+log = logging.getLogger("openpype.hosts.webpublisher")
+
+HOST_DIR = os.path.dirname(os.path.abspath(
+    openpype.hosts.webpublisher.__file__))
+PLUGINS_DIR = os.path.join(HOST_DIR, "plugins")
+PUBLISH_PATH = os.path.join(PLUGINS_DIR, "publish")
+LOAD_PATH = os.path.join(PLUGINS_DIR, "load")
+CREATE_PATH = os.path.join(PLUGINS_DIR, "create")
+
+
+def application_launch():
+    pass
+
+
+def install():
+    print("Installing Pype config...")
+
+    pyblish.register_plugin_path(PUBLISH_PATH)
+    avalon.register_plugin_path(avalon.Loader, LOAD_PATH)
+    avalon.register_plugin_path(avalon.Creator, CREATE_PATH)
+    log.info(PUBLISH_PATH)
+
+    io.install()
+    avalon.Session["AVALON_APP"] = "webpublisher"  # because of Ftrack collect
```

Comment:
Rather set `os.environ["AVLAON_APP"]` before `io.install()`. It is used as environ on multiple places.

Reply:
I just found out you're setting the environment before this line so it's not needed at all. Here: https://github.com/pypeclub/OpenPype/pull/1876/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.py#diff-cca2d2ea968f53e867594e32b799672210e1a4ff7ebd1c8d93540b1323c8d2d6R161

Reply:
Yes, here it only because setting to Session, as we sometimes use os.environ["AVALON_APP"] sometimes `avalonSession["AVALON_APP"]`. I dont think I have `avalon.Session` in the other place.

Reply:
During `io.install()` are all these keys converted from `os.environ` to `avalon.api.Session`. So you are overriding already set key.